db/migrate/20210811141323_add_indexes_on_lighthouse_samples.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Add indexes for lighthouse samples to assist in ETL failures where
```

Comment:
I would just add something like "Cotrack product ETL failures" - for the future when we have no idea what it's on about :)

Reply:
👍